packages/babel-plugin-proposal-optional-chaining/test/fixtures/general/if-optimisation/output.js

```diff
@@ -0,0 +1,17 @@
+var _a3, _a3$b, _a4, _a4$b, _a5, _a5$b;
+
+if (a && a.b && a.b.c) {
```

Comment:
I don't think this is a worthwhile optimization, this is exactly the bug we're trying to avoid with the nullish check!